azure-pipelines.yml

```diff
@@ -18,6 +18,7 @@ jobs:
   parameters:
     repo: radialgm
     enigmaref: $(Build.SourceBranch)
+    ref: refs/pull/138/merge # Use this for testing a co-dependent rgm pr but comment this before merging 
```

Comment:
reminder to remove this before merging. and to update submodule and merge RadialGM/#138 after merging this